### PR TITLE
Node Animation Tutorial

### PR DESCRIPTION
**Pull request by sercero**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,13 +24,14 @@ Note on Windows: you have to set the correct path to `OGRETOOLS_XML_CONVERTER` i
 After installing the addon enable it in Blender from `User Preferences > Add-Ons > Import-Export` (CTRL+ALT+U). Search for `ogre` and check the box on the right. Remember to save as default if you want the addon to be enabled after you exit your Blender.  
 Integrated help docs will be shown in the upper right hand toolbar, replacing blender's normal `Help` menu, read them for assistance.
 
-The following versions of blender have been tested. It might also work with older versions of Blender.
+The following versions of blender have been tested.
 
-* Blender 2.7x
```
Comment (paroj) [inline README.md:28]: did you want to target master with this?

File: NodeAnimations.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+	...
+    for (auto animationState : mSceneMgr->getAnimationStates())
+    {
+		std::cout << "animationState: " << animationState.second->getAnimationName() << std::endl;
+		animationState.second->setEnabled(true);
+		animationState.second->setLoop(true);
+    }
+}
+
+bool MyOgreApp::frameRenderingQueued(const Ogre::FrameEvent& evt)
```
Comment (paroj) [inline NodeAnimations.md:161]: this is not the best thing to do. Instead this should be used
```cpp
        auto& controllerMgr = ControllerManager::getSingleton();
        controllerMgr.createFrameTimePassthroughController(AnimationStateControllerValue::create(animState, true));
```

File: README.md (hunk 1 above)
Comment (sercero) [reply README.md:28]: Thanks for catching that one, I forgot that the READMEs are not exactly the same between 2.7 and master

File: NodeAnimations.md (hunk 2 above)
Comment (sercero) [reply NodeAnimations.md:161]: How can you have this massive API loaded in your head? LoL

I wasn't aware of that class, will look into it.

**Pull request merged**